src/packagedcode/pypi.py

```diff
@@ -112,6 +113,7 @@ def parse(location):
         file_name = fileutils.file_name(location)
         parsers = {
             'setup.py': parse_setup_py,
+            '.txt': parse_requirements_txt,
```

Comment:
See comment above. I am skeptical with this approach (but I could be wrong)

@pombredanne thoughts?